src/py/flwr/simulation/run_simulation.py

```diff
@@ -163,7 +163,7 @@ def server_th_with_start_checks(  # type: ignore
             "server_app_attr": server_app_attr,
             "loaded_server_app": server_app,
             "driver": driver,
-            "server_app_dir": server_app_dir,
+            "server_app_dir": app_dir,
```

Comment:
Should this be renamed as well?
```suggestion
            "app_dir": app_dir,
```